apps/test-bundles/webpackUtils.js

```diff
@@ -29,7 +30,7 @@ function createWebpackConfig(entries) {
           modules: true,
 
           builtAt: false,
-          outputPath: false,
+          .../** @type {*} */ ({ outputPath: false }),
```

Comment:
- can we use `any` rather than `*` ( which is the same in JSDoc context )
- why is this needed ? to allow proper assignment which is not valid from type perspective ? if that's the case we probably should assert whole assignment

Reply:
I'm actually not sure if `outputPath` is a valid option here--probably either it was valid in a previous version but isn't now, or is undocumented/missing from types. I was just changing the types to get rid of a warning in the editor and didn't want to dig into the actual options used. (All the other options appear to be valid, hence not casting the whole thing.)

Reply:
Okay looks like `outputPath` is a valid option, just missing from `webpack-bundle-analyzer`'s types. https://webpack.js.org/configuration/stats/#statsoutputpath